fa20/syllabus.yml

```diff
@@ -21,12 +21,12 @@
     authors: []
     filename: meeting01
     cover-image: ''
-    tags: []
+    tags: ["ai", "machine learning"]
     room: Online
     abstract: >-
-      We're filling this out!
+      This weeks paper we're going over a super fun application of ML in Video Games! Well see how AI/ML techniques can help enhance games like Legend of Zelda and provide limitless engaging content
   optional:  # All `optional` keys are enumerated in the Documentation
-    papers:
+    papers: https://arxiv.org/pdf/2001.05065.pdf
```

Comment:
This should be a dict.

```yaml
papers:
  <custom-pdf-name>: <url>
```

It's on my list to make these more verbose, sorry.